.github/ISSUE_TEMPLATE/question-query-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Question/Query template
+about: Use this template to file a question/query for the SDK team
+title: "[QUERY]"
+labels: ''
+assignees: ''
+
+---
+
+**Query/Question**
```

Comment:
Honestly, I'd add a link to Stack Overflow here that includes a notification-hubs tag that shows that we're responsive there. SO is a much better platform that GH for question/query type stuff.

Reply:
Here's the link I'd include: https://stackoverflow.com/questions/tagged/azure-notificationhub/

Reply:
Fixed in 4a6b0101d68a2565b9b7a99de9230357d4eb6d46